### PR TITLE
ExpandableSection - add minHeight

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
ExpandableSection - add minHeight

## Changelog
ExpandableSection - add minHeight

## Additional info
Ticket 4116

### CODE REVIEW

File: src/components/expandableSection/index.tsx [hunk 1]
```diff
@@ -25,6 +26,12 @@ export type ExpandableSectionProps = {
    * action for when pressing the header of the expandableSection
    */
   onPress?: () => void;
+  /**
+   * Set a minimum height for the expandableSection
+   * If the children height is less than the minHeight, the expandableSection will collapse to that height
+   * If the children height is greater than the minHeight, the expandableSection will result with only the children rendered
```
Comment (Inbal-Tish) [inline src/components/expandableSection/index.tsx:32]: No clear "will result with only the children rendered". Do you mean "no height limit will apply"?

File: demo/src/screens/componentScreens/ExpandableSectionScreen.tsx [hunk 2]
```diff
@@ -1,119 +1,132 @@
...
+    return this.renderHeader('Where are you located?', false, {disabled: true, showInfo: true});
   }
 
   render() {
-    const {expanded, top} = this.state;
-
+    const {minHeight} = this.state;
     return (
       <ScrollView>
-        <View row center margin-20>
```
Comment (Inbal-Tish) [inline demo/src/screens/componentScreens/ExpandableSectionScreen.tsx:93]: Missing our usual title "ExpandableSection"

File: src/components/expandableSection/index.tsx [hunk 3]
```diff
@@ -38,25 +45,39 @@ export type ExpandableSectionProps = {
...
   };
 
-  const expandableStyle = useAnimatedStyle(() => {
-    animatedHeight.value = expanded ? withTiming(height) : withTiming(0);
+  const animateHeight = useCallback((shouldAnimate = true) => {
+    const collapsedHeight = Math.min(minHeight ?? 0, height);
+    const toValue = expanded ? height : collapsedHeight;
+    animatedHeight.value = shouldAnimate ? withTiming(toValue) : toValue;
+  },
+  [animatedHeight, expanded, height, minHeight]);
```
Comment (Inbal-Tish) [inline src/components/expandableSection/index.tsx:66]: no need to break the line here

File: src/components/expandableSection/index.tsx (hunk 1 above)
Comment (M-i-k-e-l) [reply src/components/expandableSection/index.tsx:32]: Added `(sectionHeader will not be rendered)`

File: src/components/expandableSection/index.tsx (hunk 3 above)
Comment (M-i-k-e-l) [reply src/components/expandableSection/index.tsx:66]: This is how prettify is indenting for me

File: demo/src/screens/componentScreens/ExpandableSectionScreen.tsx (hunk 2 above)
Comment (M-i-k-e-l) [reply demo/src/screens/componentScreens/ExpandableSectionScreen.tsx:93]: Some screens do not have that and we have it in the RNN title

Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/ExpandableSectionScreen.tsx:93]: I know, but it's strange the the title is "Minimum Height"...

**Pull request merged**